src/client/pydaos/raw/conversion.py

```diff
@@ -29,3 +32,19 @@ def str_to_c_uuid(uuidstr):
     cuuid = (ctypes.c_ubyte * 16)()
     c_uuid(puuid, cuuid)
     return cuuid
+
+
+def c_err_to_str(err_num):
```

Comment:
We're trying to move away from `pydaos.raw`, is there a need to extend it like this?

Reply:
I did not find another way to convert an error code into its string identifier.
If such function exists to convert an error code into its error string identifier, it will be totally fine for me to use it instead of this new one.
If it does not exist, I have also no problem to move it in a different place.

Reply:
Even though we don't want to use `pydaos.raw` anymore, I think this is the right place for this function. At least until we have a "proper" pydaos API for this sort of thing

Reply:
I think pydaos has a err_to_str function.

Reply:
That's what this function uses

Reply:
So it does.

Perhaps the test should call pydaos directly rather than pydaos.raw?  Otherwise it's just one more thing for us to unwind later on.

Reply:
The idea was to hide some magic trick of low level `pydaos_shim.err_to_str()` function such as `DAOS_MAGIC` constant.
However, if you think that it is better to not add a wrapper, I could remove it and directly use `pydaos_shim.err_to_str()` function.

Reply:
You're right, that function is *weird*.  It probably doesn't need `DAOS_MAGIC` at all, in fact only the `daos_init` call does.  Let's clean that up in a different ticket and move forward with this PR as it is.

Reply:
I've created https://daosio.atlassian.net/browse/DAOS-14572 and assigned it to myself to improve pydaos here.